### PR TITLE
Enhance entity sprites

### PR DESCRIPTION
**Pull request by GTcreyon**
# Description of changes
Updates coin and death animations to use new sprites. All enemies now make a small smoke poof on death.

### CODE REVIEW

**Review by RandomCatDude — APPROVED** [review]
looks good to me.
the smoke puff feels a little strange, but im gonna put that on the lack of sfx for it, and that's defo something that could be handled later in a separate PR dedicated to audio

**Comment by Koopa1018** [comment]
> the smoke puff feels a little strange, but im gonna put that on the lack of sfx for it

Noted. I'll add that to #35.

File: classes/entity/enemy/bobomb/bobomb.tscn [hunk 1]
```diff
@@ -93,6 +93,7 @@ extents = Vector2( 1, 1 )
 collision_layer = 4
 script = ExtResource( 7 )
 _sprite_path = NodePath("Sprites/Base")
+make_smoke = false
```
Comment (Koopa1018) [inline classes/entity/enemy/bobomb/bobomb.tscn:96]: Might be wise to call this `make_death_smoke` (or just `death_smoke`) so it's clear when the smoke in question appears.

Comment (GTcreyon) [reply classes/entity/enemy/bobomb/bobomb.tscn]: Done.

**Pull request merged**